supplier_app/views.py

```diff
@@ -254,6 +257,10 @@ def approve_taxpayer(self, taxpayer_id, request=None):
     taxpayer = TaxPayer.objects.get(pk=taxpayer_id)
     taxpayer.approve_taxpayer()
     taxpayer.save()
+    subject = email_notifications['taxpayer_approval']['subject']
+    body = email_notifications['taxpayer_approval']['body']
+    emails = get_user_emails_by_tax_payer_id(taxpayer_id)
+    send_email_notification(subject, body, emails)
```

Comment:
It looks like you are sending emails in multiple flows, think and discuss the async implementation as a Team!

Reply:
Created a spike to investigate and discuss about it to implement it on the project.
[EDA-672](https://jira.evbhome.com/browse/EDA-672)

Reply:
Nice